src/electron/electron/backup_file.cljs

```diff
@@ -1,6 +1,6 @@
 (ns electron.backup-file
   (:require [clojure.string :as string]
-            ["path" :as path]
+            ["path" :as node-path]
```

Comment:
I would rather have not renamed all the electron namespaces but I had to do it since electron namespaces are coupled to the frontend ones under src/. If we move electron to its own dir then it can have its own tests and coding standards. If we're ok with doing that in a follow up PR, then I can revert the electron renaming commit